pmlc/conversion/pxa_to_affine/pxa_to_affine.cc

```diff
@@ -25,6 +25,13 @@ namespace stdx = dialect::stdx;
 
 namespace {
 
+static void splitAffineMaps(AffineMap from, SmallVectorImpl<AffineMap> &into) {
```

Comment:
IMHO, this function can be moved to `pmlc/util/util.h`, since it is used in several places.